1.19_linux/net/cgo_linux.go.patch

```diff
@@ -0,0 +1,4 @@
+//--from
+//go:build !android && cgo && !netgo
+//--to
+//go:build ignore
```

Comment:
Why do we change files in `net` in this PR?

Reply:
OK so please update the description of this PR. I thought this PR was only for signals.

Reply:
Done!